TESTING.md

```diff
@@ -0,0 +1,182 @@
+# Testing 
+
+This document describes pointers and notes for testing the GPII infrastructure in different contexts.
+
+# End-to-End Manual Testing Tutorial
+
+This section contains a tutorial to test the frontend GPII application against the backend. Relevant calls against the backend are noted after each action.
+
+## Testing key-in/key-out and basic settings
+
+1. Obtain a copy of the gpii/windows repository:
+
+```bash
+$ git clone git@github.com:GPII/windows.git
+```
+
+2. Use vagrant to spin up the instance.
+
+```bash
+$ cd windows
+
+$ vagrant up
+```
+
+3. Once the windows image is spun up, open a command prompt within the virtual machine and spin up the components locally.
+
+```shell
+$ cd C:\Vagrant
+
+$ SET GPII_CLOUD_URL=http://flowmanager.<CLUSTER_NAME>
+
+$ SET NODE_ENV=gpii.config.untrusted.development
+
+$ node gpii
+
+```
+
+The above will bring up a local flowmanager and point it to the specified cloud based flowmanager.
+
+4. Once the node process is started, create a new command prompt and login:
```

Comment:
Tip: You can do exactly the same from the "gpii-app" section below and get rid of the windows cloning stuff. We only deliver the GPII through "gpii-app", which includes the windows side of the GPII (this is, the windows repo).
For this, I recommend to only document the instructions to perform the testing from universal and gpii-app.

Reply:
Sounds like excellent advice. I'll update this document so that the instructions work off of `gpii-app`.

Reply:
Looks like this comment was never addressed.